pymess/backend/dialer/daktela.py

```diff
@@ -82,12 +68,8 @@ def _update_dialer_states(self, messages):
             state_mapped = self.config.STATES_MAPPING[resp_message_state]
             message_error = resp_json['error'] if len(resp_json['error']) else None
 
-            if message.is_autodialer:
-                is_final_state, message_state = self._get_autodialer_message_states(resp_json, state_mapped,
-                                                                                    resp_message_state)
-            else:
-                message_state = state_mapped
-                is_final_state = resp_json['result']['action'] == '5'
+            is_final_state = resp_json['result']['action'] == '5'
+            message_state = state_mapped
```

Comment:
This variable i no longer needed. Use `state_mapped` directly.

Reply:
You are right. Fixed.